definitions/npm/axios_v0.19.x/test_axios_v0.19.x.js

```diff
@@ -0,0 +1,133 @@
+// @flow
+import { describe, it } from 'flow-typed-test';
+import axios from 'axios';
+import type { $AxiosXHR, CancelTokenSource, Cancel, AxiosPromise } from 'axios';
+(axios.get('foo'): Promise<*>);
```

Comment:
For extra credit can you start using describe/it blocks?

Reply:
Also can you move it under the version-specific folder? The tests sometimes vary based on flow version.

Reply:
Done